bin/stencil-start

```diff
@@ -147,116 +135,111 @@ Wreck.get(
-                    return console.error(e);
-                }
+            Bs.reload();
+        }
+    });
 
-                Bs.reload();
-            }
-        });
+    Bs.watch('.config/storefront.json', (event, file) => {
+        if (event === 'change') {
+            console.log("storefront json changed");
+            Bs.emitter.emit("storefront_config_file:changed", {
+                event: event,
+                path: file,
+                namespace: ""
+            });
+            Bs.reload();
+        }
+    });
 
-        Bs.watch('.config/storefront.json', (event, file) => {
-            if (event === 'change') {
-                console.log("storefront json changed");
-                Bs.emitter.emit("storefront_config_file:changed", {
-                    event: event,
-                    path: file,
-                    namespace: ""
-                });
-                Bs.reload();
+    Bs.watch(templatePath, {ignoreInitial: true}, () => {
+        assembleTemplates(templatePath, (err, results) => {
+            if (err) {
+                return console.error(err);
             }
-        });
 
-        Bs.watch(templatePath, {ignoreInitial: true}, () => {
-            assembleTemplates(templatePath, (err, results) => {
-                if (err) {
-                    return console.error(err);
-                }
-
-                try {
-                    new Cycles(results).detect();
-                } catch (e) {
-                    console.error(e);
-                }
-            })
-        });
+            try {
+                new Cycles(results).detect();
+            } catch (e) {
+                console.error(e);
+            }
+        })
+    });
 
-        if (buildConfig.watchOptions && buildConfig.watchOptions.files) {
-            watchFiles = buildConfig.watchOptions.files;
-        }
+    if (buildConfig.watchOptions && buildConfig.watchOptions.files) {
+        watchFiles = buildConfig.watchOptions.files;
+    }
 
-        if (buildConfig.watchOptions && buildConfig.watchOptions.ignored) {
-            watchIgnored = buildConfig.watchOptions.ignored;
-        }
+    if (buildConfig.watchOptions && buildConfig.watchOptions.ignored) {
+        watchIgnored = buildConfig.watchOptions.ignored;
+    }
 
-        // check for tunnel argument
-        if (typeof Program.tunnel === 'undefined') { tunnel = false; }
-        else if (Program.tunnel === true)  { tunnel = true; }
-        else { tunnel = Program.tunnel; }
-
-        Bs.init({
-            open: !!Program.open,
-            port: browserSyncPort,
-            files: watchFiles.map(val => Path.join(themePath, val)),
-            watchOptions: {
-                ignoreInitial: true,
-                ignored: watchIgnored.map(val => Path.join(themePath, val)),
-            },
-            proxy: "localhost:" + stencilServerPort,
-            tunnel: tunnel,
-        });
+    const tunnel = typeof Program.tunnel === 'undefined'
```

Comment:
we can move this to the top where we define other constants 🍹

Reply:
yeah

Reply:
updated